pages/linux/hddtemp.md

```diff
@@ -0,0 +1,36 @@
+# hddtemp
+
+> Display temperature of HDD via S.M.A.R.T.
+> More information: <https://manned.org/hddtemp>.
+
+- Display temperature of a specific drive:
+
+`hddtemp {{type:}}{{/dev/sdX}}`
```

Comment:
```suggestion
`hddtemp {{type}}:{{/dev/sdX}}`
```
I assume the colon is always there
Change it for all other cases too if that is the case